agent_testing/Alice.py

```diff
@@ -0,0 +1,108 @@
+"""Alice return route capable agent.
+
+This server is meant to interpret the messages of the
+corresponding Bob client and carry out their requests.
+"""
+
+import hashlib
+import re
+import os
+from aiohttp import web
+from aries_staticagent import StaticConnection, Target, crypto, utils, route
+
+
+def main():
+
+    """Start a server with a static connection."""
+    their_vk, _ = crypto.create_keypair(seed=hashlib.sha256(b"client").digest())
+    conn = StaticConnection.from_seed(
+        hashlib.sha256(b"server").digest(), Target(their_vk=their_vk)
+    )
+
+    my_list = [{"name": "item1", "features": "feature1"}]
+
+    def delete_item(item):
+        # Deletes an item from the given list
+
+        del my_list[int(item) - 1]
+
+        response = {"Success!": "Item number " + str(item) + " deleted!"}
+        return response
+
+    """Make handlers for our three message types."""
+
+    @conn.route("https://didcomm.org/basicmessage/1.0/add")
```

Comment:
Typically, decorators are placed on the line immediately above the method they're decorating. Please move this decorator (and all others in these modules) to immediately above the decorated method. (i.e. move this line to line 36)